### PR TITLE
Fix localization failure

### PR DESCRIPTION
**Pull request by buyaa-n**
I cannot push to the cristianosuzuki77/master directly, @cristianosuzuki77 could you merge this PR, or maybe you prefer this merged to `dotnet/roslyn-analyzers` instead? CC @mavasani

### CODE REVIEW

**Comment by mavasani** [comment]
@buyaa-n You may also consider just targeting master in this PR, and once this is merged, I can re-run tests on other PR and then merge. Thanks again for super quick workaround!!

File: src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs [hunk 1]
```diff
@@ -1839,9 +1840,11 @@ public class OsDependentClass
             {
                 await VerifyAnalyzerAsyncCs(source,
                     VerifyCS.Diagnostic(PlatformCompatibilityAnalyzer.OnlySupportedCsReachable).WithSpan(9, 32, 9, 54).
-                    WithArguments("OsDependentClass", $"'{platform}' {version} and later", $"'{suppressingPlatform}' {suppressingVersion} and later"),
+                    WithArguments("OsDependentClass", string.Format(CultureInfo.InvariantCulture, MicrosoftNetCoreAnalyzersResources.PlatformCompatibilityVersionAndLater, platform, version),
+                    string.Format(CultureInfo.InvariantCulture, MicrosoftNetCoreAnalyzersResources.PlatformCompatibilityVersionAndLater, suppressingPlatform, suppressingVersion)),
```
Comment (mavasani) [inline src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs:1844]: Probably worth writing a private helper for this which takes just the format arguments? I'll leave it to you to decide..

**Comment by buyaa-n** [comment]
> @buyaa-n You may also consider just targeting master in this PR, and once this is merged, I can re-run tests on other PR and then merge. Thanks again for super quick workaround!!

Thanks, created a PR to dotnet repo, closing this one

**Comment by cristianosuzuki77** [comment]
Apologies, was OOO

**Pull request closed without merging**